Code/Editor/ViewPane.cpp

```diff
@@ -424,18 +588,68 @@ void CLayoutViewPane::OnMenuBindingHook()
         m_menuManagerInterface->AddActionToMenu(ViewportHelpersMenuIdentifier, "o3de.action.view.toggleIcons", 200);
         m_menuManagerInterface->AddActionToMenu(ViewportHelpersMenuIdentifier, "o3de.action.view.toggleSelectedEntityHelpers", 300);
     }
+
+    // Size
+    {
+        m_menuManagerInterface->AddSubMenuToMenu(ViewportSizeMenuIdentifier, ViewportSizeRatioMenuIdentifier, 100);
+        {
+            for (size_t i = 0; i < ViewportRatiosCount; ++i)
+            {
+                int width = ViewportRatios[i].first;
+                int height = ViewportRatios[i].second;
+                AZStd::string actionIdentifier = AZStd::string::format("o3de.action.viewport.size.ratio[%i:%i]", width, height);
+
+                m_menuManagerInterface->AddActionToMenu(ViewportSizeRatioMenuIdentifier, actionIdentifier, 100 * (aznumeric_cast<int>(i) + 1));
```

Comment:
suggestion: In this function is it worth moving the `100`s to a constant so it's easier to know what it is referring to (be easier to change in one place in future too then 🙂)